semanticscholar/SemanticScholar.py

```diff
@@ -61,12 +46,12 @@ def timeout(self, timeout: int) -> None:
         :param int timeout:
         '''
         self._timeout = timeout
-        self._requester.timeout = timeout
+        self._AsyncSemanticScholar.timeout = timeout
 
     def get_paper(
                 self,
-                paper_id: str,
-                fields: list = None
+                *args,
+                **kwargs
```

Comment:
Thinking about the documentation, I believe we should duplicate the method signatures, including all the parameters.